vscode-ext/src/extension.ts

```diff
@@ -24,90 +48,33 @@ function getK8sApi(): CoreV1Api {
-	'Toggle outgoing traffic (current: ' + (outgoingTraffic ? 'enabled' : 'disabled') + ')',
-	'Include environment variables (current: ' + includeEnvironmentVariables + ')',
-	'Exclude environment variables (current: ' + excludeEnvironmentVariables + ')',
-	'Toggle telemetry (current: ' + telemetry + ')'];
-	vscode.window.showQuickPick(options).then(async setting => {
-		if (setting === undefined) {
-			return;
-		}
-		else if (setting.startsWith('Toggle telemetry')) {
-			globalContext.workspaceState.update('telemetry', !telemetry);
-		}
-		else if (setting.startsWith('Toggle file')) {
-			globalContext.workspaceState.update('fileOps', !fileOps);
-		}
-		else if (setting.startsWith('Toggle invalid certificates')) {
-			globalContext.workspaceState.update('invalidCertificates', !invalidCertificates);
-		}
-		else if (setting.startsWith('Toggle traffic stealing')) {
-			globalContext.workspaceState.update('trafficStealing', !trafficStealing);
-		}
-		else if (setting.startsWith('Toggle remote DNS')) {
-			globalContext.workspaceState.update('remoteDNS', !remoteDNS);
-		}
-		else if (setting.startsWith('Toggle outgoing traffic')) {
-			globalContext.workspaceState.update('outgoingTraffic', !outgoingTraffic);
-		}
-		else if (setting.startsWith('Include environment variables')) {
-			vscode.window.showInputBox({ prompt: 'Enter environment variables to include (separated by commas, * for all)', value: includeEnvironmentVariables }).then(async value => {
-				if (value === undefined) {
-					return;
-				}
-				globalContext.workspaceState.update('includeEnvironmentVariables', value);
-			});
-		}
-		else if (setting.startsWith('Exclude environment variables')) {
-			vscode.window.showInputBox({ prompt: 'Enter environment variables to exclude', value: excludeEnvironmentVariables }).then(async value => {
-				if (value === undefined) {
-					return;
-				}
-				globalContext.workspaceState.update('excludeEnvironmentVariables', value);
-			});
-		} else if (setting.startsWith('Change namespace')) {
-			const namespaces: {
-				response: any;
-				body: V1NamespaceList;
-			} = await getK8sApi().listNamespace();
-			const namespaceNames = namespaces.body.items.map(namespace => namespace.metadata!.name!);
-			vscode.window.showQuickPick(namespaceNames, { placeHolder: 'Select namespace' }).then(async namespaceName => {
-				if (namespaceName === undefined) {
-					return;
-				}
-				if (setting.startsWith('Change namespace for mirrord agent')) {
-					globalContext.workspaceState.update('agentNamespace', namespaceName);
-				} else if (setting.startsWith('Change namespace for impersonated pod')) {
-					globalContext.workspaceState.update('impersonatedPodNamespace', namespaceName);
-				}
-			});
-		}
-	});
+async function configFilePath() {
+	let vsCodeDir = VS_CODE_DIR;
+	let fileUri = vscode.Uri.joinPath(vsCodeDir, 'mirrord.+(toml|json|y?(a)ml)');
+	let files = glob.sync(fileUri.fsPath);
```

Comment:
looks like we are trying to look for the config.. can we add a short comment here